start.nginx.sh

```diff
@@ -25,4 +25,7 @@ php artisan instance:actor
 
 # Finally run everything
 php-fpm -D &
-nginx -g 'daemon off;'
+nginx -g 'daemon off;' &
+if [ "${ENABLE_HORIZON:-false}" = "true" ]; then
+    php artisan horizon
+fi
```

Comment:
Did you test this both enabled and not? I think if it's not enabled, there will be no process in the foreground and the container will terminate.

You probably just want to rearrange this such that the last execution will be nginx in the foreground...

```suggestion
if [ "${ENABLE_HORIZON:-false}" = "true" ]; then
    php artisan horizon &
fi
nginx -g 'daemon off;'
```

Reply:
i did test as written (tailing &) inside the container and it surprisingly worked, but I like your suggestion better. addressed in my latest commit